src/Send/Memo.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {HelperText, TextInput} from '../components'
+
+export const maxMemoLength = 256
+
+type Props = {
+  memo: string
+  onChangeText: (memo: string) => void
+}
+
+export const Memo = ({onChangeText, memo}: Props) => {
+  const strings = useStrings()
+
+  const showError = memo.length > maxMemoLength
+  const multilineMax = memo.length > 50
```

Comment:
arbitrary number to meet requirements

https://github.com/Emurgo/yoroi-mobile/pull/2324/files#r1105108493